home/templates/home/tags.html

```diff
@@ -0,0 +1,60 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+<html>
+    <style>
+        body {
+            margin: auto;
+            background-color: #D9B6A3;
+        }
+        form.d-flex input {
+            border: 1px solid black;
+        }
+        form.d-flex button {
+            background-color: #A9D4D9;
+            border: 1px solid black;
+        }
+        .wrapper {
+            display: grid;
+            grid-template-columns: repeat(3, 1fr);
+            gap: 20px;
+            margin-top: 20px;
+        }
+        .item-in-wrapper {
+            cursor: pointer;
+            position: relative;
+            height: 200px;
+            width: 180px;
+            text-align: center;
+        }
+    </style>
```

Comment:
Barak recommended having all the styles in a CSS file outside of the template for improving browser-side caching.
You could put the file in the static/home/CSS directory

Reply:
Thank you:) I'll add a file in the directory for my page.